e2e/cypress/tests/integration/multi_team_and_dm/existing_channel_name_spec.js

```diff
@@ -62,11 +60,7 @@ function verifyExistingChannelError(newChannelName, makePrivate = false) {
     }
 
     // Type `newChannelName` in the input field for new channel
-    cy.get('#input_new-channel-modal-name').should('be.visible').click().type(newChannelName);
-    cy.wait(TIMEOUTS.HALF_SEC);
-
-    // Click 'Create Channel' button
-    cy.findByText('Create channel').click();
```

Comment:
shouldn't we keep both scenarios? click create channel button  and enter key press?

Reply:
Sure, there are other tests that will keep using the click event. ie. https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/ui/channel.js#L26